### DISCUSSION TITLE
Buttons are unstyled and with forms plugin they are forced to be transparent

### DISCUSSION THREAD

**Discussion by OnkelTem** [tailwindlabs/tailwindcss#9959] [Help]
I wonder how to make a minimal good-looking form. 

I created a sandbox at Stackblitz, but I cannot get buttons styled in any sensible way. 

Currently, they look like text. I tried applying some global styles but they got overridden but the `forms` tailwind plugin. 

Here is my sandbox:
https://stackblitz.com/edit/react-ts-uyfwgm?file=App.tsx

As you see, the form gets styles inline and submit button isn't styled as a button at all. What do I do to get a normal styled form?

**Comment by wongjn** [comment]
To be clear, the reset styles applied to the `<button>` element comes from the [preflight core plugin](https://tailwindcss.com/docs/preflight), not `@tailwindcss/forms`. Regardless, the reason the `button` rule you defined is not applying is that one of the preflight rules has the selector, `[type="submit"]`, which matches your form's `<button>` element and overrides your `button` selector, since the former has specificity (0, 1, 0) while yours only has (0, 0, 1). You could increase the specificity of your rule, such as by using the same selector, `[type="submit"]` instead of `button`.
